internal/builders/golang/build.go

```diff
@@ -161,7 +193,7 @@ func buildGoBuildLine(ctx *context.Context, build config.Build, options api.Opti
 	// ldflags is not a repeatable flag
 	if len(build.Ldflags) > 0 {
```

Comment:
Similarly here:

```suggestion
	if len(details.Ldflags) > 0 {
```

Reply:
yes, I'm blind! hehe

will also ensure this is covered by tests